### PR TITLE
release: qase-robotframework 3.0.0b1

### PR DESCRIPTION
**Pull request by gibiw**
Updated the documentation.

### CODE REVIEW

File: qase-robotframework/README.md [hunk 1]
```diff
@@ -1,17 +1,71 @@
...
-> Publish results simple and easy.
+# [Qase TestOps](https://qase.io) Robot Framework Reporter
 
-## How to integrate
+[![License](https://lxgaming.github.io/badges/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
-```
+## Installation
+
+```sh
```
Comment (NickVolynkin) [inline qase-robotframework/README.md:7]: will it install a beta package?

File: qase-robotframework/docs/UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+-   "chunk": 100
++   "batch": {
++     "size": 100
++   }
+  }
+}
+```
+
+### Configuration
+In the latest version of `qase-robotframework` reporter, we have changed the way of configuration. The primary source of truth from now is a config file `qase.config.json` that should be located in the same directory from where you run tests. This change will allow us to add more configuration options in the future and make the configuration process more flexible. This config structure will be used for all other reporters.
```
Comment (NickVolynkin) [inline qase-robotframework/docs/UPGRADE.md:27]: More lines => better diff
```suggestion
In the latest version of `qase-robotframework` reporter, we have changed the way of configuration.
The primary source of truth from now is a config file `qase.config.json`
that should be located in the same directory from where you run tests.
This change will allow us to add more configuration options in the future
and make the configuration process more flexible.
This config structure will be used for all other reporters.
```

File: qase-robotframework/docs/UPGRADE.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Also, we have changed the names of ENV variables. You can find the list of all available options in the [configuration](../README.md#configuration) section.
```
Comment (NickVolynkin) [inline qase-robotframework/docs/UPGRADE.md:29]:
```suggestion
Also, the names of ENV variables have changed.
You can find the list of all available options in the [configuration](../README.md#configuration) section.
```

File: qase-robotframework/README.md [hunk 4]
```diff
@@ -1,17 +1,71 @@
...
+
+## Configuration
+
+Qase Robot Framework Reporter is configured in multiple ways:
+
+- using a config file `qase.config.json`
+- using environment variables
+
+Environment variables override the values given in the config file, 
+and command line options override both other values.
```
[the remark this answers is not in the document]
Comment (gibiw) [reply qase-robotframework/README.md:24]: No. We can use only the ENV variables and the config file.

File: qase-robotframework/README.md [hunk 5]
```diff
@@ -1,17 +1,71 @@
...
```
Comment (NickVolynkin) [inline qase-robotframework/README.md:24]:
```suggestion
Environment variables override the values given in the config file.
```

File: qase-robotframework/docs/CONFIGURATION.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Qase Robot Framework Plugin configuration
+
+Qase Robot Framework Reporter is configured in multiple ways:
+
+- using a config file `qase.config.json`
+- using environment variables
+
+Environment variables override the values given in the config file, 
+and command line options override both other values.
```
Comment (NickVolynkin) [inline qase-robotframework/docs/CONFIGURATION.md:9]:
```suggestion
Environment variables override the values given in the config file.
```

File: qase-robotframework/README.md (hunk 5 above)
Comment (gibiw) [reply qase-robotframework/README.md]: Fixed

File: qase-robotframework/docs/CONFIGURATION.md (hunk 6 above)
Comment (gibiw) [reply qase-robotframework/docs/CONFIGURATION.md]: Fixed

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**